caampr-backend/src/main/java/com/jelistan/caampr/lambda/adaptor/GearListRequestAdaptor.java

```diff
@@ -0,0 +1,20 @@
+package com.jelistan.caampr.lambda.adaptor;
+
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.jelistan.caampr.lambda.model.GearListRequest;
+
+/**
+ * Converts APIGatewayProxyRequestEvent (external) to GearListRequest (internal)
+ */
+
+public class GearListRequestAdaptor{
+    public GearListRequest convert(APIGatewayProxyRequestEvent requestEvent){
+        String[] tokens = requestEvent.getPath().split("/");
+        String profileId = tokens[2];
+        GearListRequest request = GearListRequest.builder()
+                .callerId("6969") //temporary hardcode until requester info is implemented
```

Comment:
just an FYI (meaning you don't have to do it here): normally, if you are leaving a stub or placeholder, add a TODO comment and create an associated issue/ticket to track that the TODO is resolved. e.g.
```
// TODO hardcoding the callerId until it's passed in for real. tracking in issue [link to issue]
```

Reply:
will do, make issue 18 for this